.github/workflows/pullrequest.yml

```diff
@@ -17,6 +17,7 @@ env:
   COMPOSE_HTTP_TIMEOUT: 180
   GITHUB_ACTIONS_CACHE_URL: https://cache.dev01.devland.is/
   SKIP_GENERATED_CACHE: ${{ contains(github.event.pull_request.labels.*.name, 'skip-generated-cache') }}
+  NX_AFFECTED_ALL: ${{ contains(github.event.pull_request.labels.*.name, 'nx-affect-all') }}
```

Comment:
Consider a name that doesn't start with `NX_`. It's "reserved" for NX internals and options.